c/src/ml-api-service-offloading.c

```diff
@@ -1184,3 +1184,20 @@ ml_service_offloading_request (ml_service_h handle, const char *key,
     nns_edge_data_destroy (data_h);
   return ret;
 }
+
+/**
+ * @brief Register new information, such as neural network models or pipeline descriptions, on a offloading server.
+ */
+int
+ml_service_offloading_request_raw (ml_service_h handle, const char *key,
+    void *data, size_t len)
+{
+  ml_tensors_data_s input;
+
+  /* Set internal data structure to send edge data. */
+  input.num_tensors = 1;
+  input.tensors[0].data = data;
+  input.tensors[0].size = len;
```

Comment:
Is there no need to set dimension and type?

Reply:
nns-edge does not check tensor shape or type. It handles data only.

Reply:
Thank you.